fs/base.py

```diff
@@ -84,7 +84,7 @@ def _method(*args, **kwargs):
 """.format(
         method.__name__
     )
-    if getattr(_method, "__doc__"):
+    if hasattr(_method, "__doc__"):
```

Comment:
Good catch. I guess this has always been True. Did flake8 flag this?

Reply:
Yeah. `getattr(thing, "some_attribute")` is no safer than `thing.some_attribute` without passing a default as the third argument since it'll still raise an `AttributeError`. I figured `hasattr` was the intent.

Reply:
Yeah that was the intent, just curious what the flake8 error was. I guess it noticed that the attribute name was a literal.